x-pack/test/security_solution_api_integration/test_suites/telemetry/tasks/ingest_pipeline_stats.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import {
+  cleanupDatastreams,
+  cleanupIngestPipelines,
+  indexRandomData,
+  launchTask,
+  randomDatastream,
+  randomIngestPipeline,
+  taskHasRun,
+  waitFor,
+} from '../../../../common/utils/security_solution';
+
+const TASK_ID = 'security:ingest-pipelines-stats-telemetry:1.0.0';
+const INGEST_PIPELINES_STATS_EBT = 'telemetry_node_ingest_pipelines_stats_event';
+
+export default ({ getService }: FtrProviderContext) => {
+  const ebtServer = getService('kibana_ebt_server');
+  const kibanaServer = getService('kibanaServer');
+  const logger = getService('log');
+  const es = getService('es');
+
+  describe('Ingest pipeline stats task telemetry.', function () {
```

Comment:
Nit: qualifying that this is for the Security telemetry task would be potentially helpful for posterity at little cost:

```suggestion
  describe('Security Telemetry - Ingest pipeline stats task.', function () {
```